python/cuxfilter/dataframe.py

```diff
@@ -22,6 +22,8 @@ class DataFrame:
     """
 
     data: Type[cudf.DataFrame] = None
+    is_graph: Type[bool] = False
```

Comment:
Why is `Type` necessary here?

Reply:
The type is not necessary here actually. The initial plan was to take is_graph as an input parameter, but that's not the case now. Will fix it

Reply:
fixed